fluss-server/src/main/java/com/alibaba/fluss/server/coordinator/event/CoordinatorEventManager.java

```diff
@@ -40,14 +44,29 @@ public final class CoordinatorEventManager implements EventManager {
     private static final String COORDINATOR_EVENT_THREAD_NAME = "coordinator-event-thread";
 
     private final EventProcessor eventProcessor;
+    private final CoordinatorMetricGroup coordinatorMetricGroup;
 
     private final LinkedBlockingQueue<CoordinatorEvent> queue = new LinkedBlockingQueue<>();
     private final CoordinatorEventThread thread =
             new CoordinatorEventThread(COORDINATOR_EVENT_THREAD_NAME);
     private final Lock putLock = new ReentrantLock();
 
-    public CoordinatorEventManager(EventProcessor eventProcessor) {
+    // metrics
+    private Counter finishedEvents;
+
+    public CoordinatorEventManager(
+            EventProcessor eventProcessor, CoordinatorMetricGroup coordinatorMetricGroup) {
         this.eventProcessor = eventProcessor;
+        this.coordinatorMetricGroup = coordinatorMetricGroup;
+        registerMetrics();
+    }
+
+    private void registerMetrics() {
+        coordinatorMetricGroup.gauge(
+                MetricNames.COORDINATOR_WAITING_TO_PROCESS_EVENT_COUNT, queue::size);
+
+        finishedEvents = new SimpleCounter();
+        coordinatorMetricGroup.meter(MetricNames.EVENT_PROCESS_RATE, new MeterView(finishedEvents));
```

Comment:
Compared to the event processing rate, a more important metric might be the event process time (`eventProcessTimeMs`), as it helps identify whether there are any time-consuming events. The event processing rate is not solely determined by the coordinator itself, making it harder to diagnose system issues using this metric alone.